Objective-C/Internal/CBLLiveQuery.mm

```diff
@@ -32,7 +32,7 @@ @implementation CBLLiveQuery
     __weak CBLQuery* _query;
     NSTimeInterval _updateInterval;
     
-    BOOL _observing, _willUpdate;
+    BOOL _observing, _willUpdate, _updating, _stopping;
```

Comment:
Again, i wonder if a single state variable wouldn't be a better way to handle this.  Would that even be possible?

Reply:
updating and stopping are independent to each other but we might be able to clean this up more.

Reply:
Track it here : https://issues.couchbase.com/browse/CBL-794